domain-http/interface/src/main/java/org/jboss/as/domain/http/server/ManagementHttpServer.java

```diff
@@ -105,6 +77,28 @@
 import org.xnio.ssl.SslConnection;
 import org.xnio.ssl.XnioSsl;
 
+import io.undertow.protocols.ssl.UndertowXnioSsl;
+import io.undertow.security.handlers.AuthenticationCallHandler;
+import io.undertow.security.handlers.AuthenticationConstraintHandler;
+import io.undertow.security.handlers.SinglePortConfidentialityHandler;
+import io.undertow.server.HttpHandler;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.server.RenegotiationRequiredException;
+import io.undertow.server.SSLSessionInfo;
+import io.undertow.server.handlers.BlockingHandler;
+import io.undertow.server.handlers.CanonicalPathHandler;
+import io.undertow.server.handlers.ChannelUpgradeHandler;
+import io.undertow.server.handlers.PathHandler;
+import io.undertow.server.handlers.ResponseCodeHandler;
+import io.undertow.server.handlers.SetHeaderHandler;
+import io.undertow.server.handlers.cache.CacheHandler;
+import io.undertow.server.handlers.cache.DirectBufferCache;
+import io.undertow.server.handlers.error.SimpleErrorPageHandler;
+import io.undertow.server.handlers.resource.ResourceManager;
+import io.undertow.server.protocol.http.HttpOpenListener;
+import io.undertow.util.Headers;
+import io.undertow.util.Methods;
```

Comment:
These have been at the bottom for quite a while now - I have to fight hard to resist the urge to automatically re-order them ;-)